packages/core/admin/admin/src/hooks/useEnterprise/useEnterprise.js

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+
+import { useCallbackRef } from '@strapi/helper-plugin';
+
+function isEnterprise() {
+  return window.strapi.isEE;
+}
```

Comment:
I don't mind. I think it is used 3 times and I found it very readable in case anyone wants to find out, how we determine the current project is running in EE mode, but I can also inline it. No feelings :p

Reply:
That works for me, I don't really have a strong opinion was just curious about the decision or if you meant to export it :)